src/PlatformInstaller/Views/SelectItemsViewModel.cs

```diff
@@ -43,7 +43,7 @@ protected override void OnInitialize()
     {
         LoadingVisibility = Visibility.Visible;
         base.OnInitialize();
-        Task.Run(() => { Init(); });
+        _ = Task.Run(() => { Init(); });
```

Comment:
Dropped task, but I don't think it matters, since this whole thing is deprecated anyway.